client/src/Components/BigLabel/BigLabel.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Story } from '@storybook/react';
+
+import { BigLabel, BigLabelComponentProps } from './BigLabel';
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
+  title: 'BigLabel',
+  component: BigLabel,
+  decorators: [(Story: any) => <Story />],
+};
+
+const Template: Story<BigLabelComponentProps> = (args) => <BigLabel {...args} />;
+
+export const PrimaryBigLabel = Template.bind({});
```

Comment:
Te same komentarze co u Sebastiana ;p
Zerknij tam i zmień tak samo :D

Reply:
poprawione, mam nadzieję, że o to chodziło...